pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -1,15 +1,17 @@
 {
-  lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonOlder,
-  pytestCheckHook,
+  fetchpatch,
   gdcm,
+  lib,
   nibabel,
   numpy,
+  pillow,
   pydicom,
-  pylibjpeg,
   pylibjpeg-libjpeg,
+  pylibjpeg,
+  pytestCheckHook,
+  pythonOlder,
   scipy,
   setuptools,
```

Comment:
```nix
{
  lib,
  buildPythonPackage,
  fetchFromGitHub,
  fetchpatch,

  # build-system
  setuptools

  # dependencies
  gdcm,
  nibabel,
  numpy,
  pydicom,
  scipy,

  # tests
  pillow,
  pylibjpeg,
  pylibjpeg-libjpeg,
  pytestCheckHook,
}
```

Reply:
I think that's all of them; just split your imports into such groups and make sure to keep them sorted